stage2/tasks/code-review/MentorEvaluationCriteria.md

```diff
@@ -0,0 +1,43 @@
+# Mentor Evaluation Criteria for Code Review Task
+
+## Introduction
+
+This document outlines the evaluation criteria for mentors assessing the code review task. The scoring is based on the student's ability to effectively identify issues, provide suggestions, and maintain professional communication.
+
+## Evaluation Criteria
+
+### 1. Identification of Code Issues (Total: 50 points)
+
+- **2 points for each identified issue**.
+- Maximum of 50 points for this criterion.
+- Focus on identifying inefficient code, poor practices, or errors.
+
+### 2. Providing Suggestions (Total: 15 points)
+
+- **3 points for each comment containing suggestions**.
+- Maximum of 15 points for this criterion.
+- Emphasis on actionable and constructive suggestions.
+
+### 3. Comment Placement Accuracy (10 points)
+
+- **10 points if all comments are accurately placed** relevant to the specific lines of code they address.
```

Comment:
**10 points if all comments are accurately placed**, _meaning they should be directly relevant to the specific lines of code they address._